sorc/link_workflow.sh

```diff
@@ -275,6 +275,28 @@ if [[ -d "${HOMEgfs}/sorc/gdas.cd/build" ]]; then
   done
 fi
 
+#------------------------------
+#-- Link the JEDI/SOCA experimental observations
+#------------------------------
+case "${machine}" in
+  "wcoss2")   EXPOBS_DIR="" ;;
+  "hera")     EXPOBS_DIR="/scratch1/NCEPDEV/global/glopara/data/experimental_obs" ;;
+  "orion")    EXPOBS_DIR="/work/noaa/global/glopara/data/experimental_obs" ;;
+  "hercules") EXPOBS_DIR="/work/noaa/global/glopara/data/experimental_obs" ;;
+  "jet")      EXPOBS_DIR="" ;;
+  "s4")       EXPOBS_DIR="" ;;
+  "gaea")     EXPOBS_DIR="" ;;
+  "noaacloud") EXPOBS_DIR="" ;;
+  *)
+    echo "FATAL: Unknown target machine ${machine}, couldn't set EXPOBS_DIR"
+    exit 1
+    ;;
+esac
```

Comment:
I have a fork branch with similar but differently done updates to handle the path to the `experimental_obs` folder. Please see the following diff and consider including my changes instead of the update you have in `sorc/link_workflow.sh` and `DMPDIR` in `parm/config/gfs/yaml/defaults.yaml`:
https://github.com/NOAA-EMC/global-workflow/compare/develop...KateFriedman-NOAA:global-workflow:feature/experimental_obs_path
I'm happy to send a PR into the branch for this PR if you are good with the updates. My updates will resolve https://github.com/NOAA-EMC/global-workflow/issues/2683.

Reply:
A pr into my branch sounds good to me @KateFriedman-NOAA , thanks.

Reply:
Great, please see https://github.com/guillaumevernieres/global-workflow/pull/8. Thanks!